sql/sql_reload.cc

```diff
@@ -216,6 +216,7 @@ bool reload_acl_and_cache(THD *thd, unsigned long long options,
               !thd->mdl_context.has_locks() ||
               thd->handler_tables_hash.records ||
               thd->ull_hash.records ||
+              thd->mdl_backup_lock ||
```

Comment:
Move the test of mdl_backup_lock one line down to be close to thd->current_backup_stage as these both test are related to backup

Reply:
Moved as suggested.